app/models/index.js

```diff
@@ -11,7 +11,8 @@ const sequelize = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.p
 fs.readdirSync(__dirname)
   .filter(file => file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js')
   .forEach(file => {
-    const model = sequelize.import(path.join(__dirname, file));
+    // eslint-disable-next-line global-require
+    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
```

Comment:
What is this change for?

Reply:
Because in Sequelize v6.x it is deprecated to do `sequelize.import`.

Source: https://sequelize.org/master/manual/models-definition.html

Reply:
> What is this change for?

@dbenavidesv  Because in Sequelize v6.x it is deprecated to do sequelize.import.

Source: https://sequelize.org/master/manual/models-definition.html